.github/workflows/tests.yml

```
@@ -42,3 +43,26 @@ jobs:
     - name: Test with pytest
       run: |
         python3 -m pytest -v
+
+  publish:
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+    needs: tests
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
```

Looks like there are new versions of these now:

```suggestion
        uses: actions/checkout@v4

      - name: Set up Python 3.10
        uses: actions/setup-python@v5
```

---

Ah good catch, fixed in 9f1ae20.